### PR TITLE
Details

### PR DESCRIPTION
**Pull request by hedita**

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -32,13 +30,39 @@ body {
...
   background-color: #F4F4F4;
-  border: 4px solid red;
+  padding: 50px 50px;
+}
+
+.title-second-section {
+  background-color: red;
+  color: #FFFFFF;
+  display: inline-block;
+  padding: 2px 8px 2px 8px;
```
Comment (takrami) [inline style.css]: You can use the shorthand for it. padding: 2px 8px;  the first one is for top and bottom and the second is for right and left. when the value for top and bottom and also left and right are the same, you should use it in that way.

File: style.css [hunk 2]
```diff
@@ -32,13 +30,39 @@ body {
...
+  color: #FFFFFF;
+  display: inline-block;
+  padding: 0 8px 0 8px;
+  font-weight: bold;
+}
+
+.first-text {
   background-color: #F4F4F4;
-  border: 4px solid red;
+  padding: 50px 50px;
```
Comment (takrami) [inline style.css]: You can use the short format for it. When the all four values are the same you can write it like this: padding 50px.

File: style.css [hunk 3]
```diff
@@ -32,13 +30,39 @@ body {
...
 .first-section {
+  text-align: center;
+  padding: 50px 50px;
+}
+
+.title-first-section {
+  background-color: red;
+  color: #FFFFFF;
+  display: inline-block;
+  padding: 0 8px 0 8px;
```
Comment (takrami) [inline style.css]: You can use the short format for it.

File: style.css [hunk 4]
```diff
@@ -32,13 +30,39 @@ body {
   margin: 0 auto;
 }
 
+.text-header {
+  text-align: center;
+}
+
 .first-section {
+  text-align: center;
+  padding: 50px 50px;
```
Comment (takrami) [inline style.css]: You can use the short format for it.

**Review by takrami — CHANGES_REQUESTED** [review]

File: style.css (hunk 1 above)
Comment (hedita) [reply style.css]: Done.

File: style.css (hunk 3 above)
Comment (hedita) [reply style.css]: Done.

File: style.css (hunk 4 above)
Comment (hedita) [reply style.css]: Fixed.

File: style.css (hunk 2 above)
Comment (hedita) [reply style.css]: Done.

**Pull request merged**